src/main/java/com/ctc/wstx/dtd/MinimalDTDReader.java

```diff
@@ -320,6 +320,7 @@ protected void skipCommentContent()
                 if (c == '-') {
                     return;
                 }
+                if (c == '\n') skipCRLF(c);
```

Comment:
I think that a better fix would be to change following line's test from `else if` to `if`, so both \r and \d would be checked.

Reply:
Changed as per above suggestion.